tests/virt_autotest/virt_utils.pm

```diff
@@ -198,4 +217,59 @@ sub clean_up_red_disks {
     diag("Debug info: Disks and File Systems Overview:\n $disks_fs_overview");
 }
 
+sub lpar_cmd {
+    my ($cmd, $mesg, $args) = @_;
+    die 'Command not provided' unless $cmd;
+    my $mesg = shift;
+
+    $args->{ignore_return_code} ||= 0;
+    my $ret = console('svirt')->run_cmd($cmd);
+    if ($ret == 0){
+        diag "Command $mesg on S390X LPAR returned: SUCESS";
```

Comment:
If you want diagnostic msg from openqa webui, then use record_info api can help.

Reply:
yes, try to use record_info api  as your mentioned, It does work to me.

Will add this record_info api.